test/Unit/PowerShell/PowerShellManagerTests.cs

```diff
@@ -47,7 +47,7 @@ static TestUtils()
         // This is to guarantee that the well known paths are setup before calling the constructor of PowerShellManager.
         internal static PowerShellManager NewTestPowerShellManager(ConsoleLogger logger, PowerShell pwsh = null)
         {
-            return pwsh != null ? new PowerShellManager(logger, pwsh) : new PowerShellManager(logger);
+            return pwsh != null ? new PowerShellManager(logger, pwsh) : new PowerShellManager(logger, id: 2);
```

Comment:
Is the `2` here significant or is it a random number?

Reply:
It's a random number :)